scripts/ethermint-devnet.yaml

```diff
@@ -0,0 +1,56 @@
+dotenv: .env
+ethermint_9000-1:
+  cmd: ethermintd
+  start-flags: "--trace"
+  app-config:
+    minimum-gas-prices: 0basephoton
+    index-events:
+      - ethereum_tx.ethereumTxHash
+    json-rpc:
+      address: "0.0.0.0:{EVMRPC_PORT}"
+      ws-address: "0.0.0.0:{EVMRPC_PORT_WS}"
+      api: "eth,net,web3,debug"
+  validators:
+    - coins: 1000000000000000000stake,10000000000000000000000basephoton
+      staked: 1000000000000000000stake
+      mnemonic: ${VALIDATOR1_MNEMONIC}
+    - coins: 1000000000000000000stake,10000000000000000000000basephoton
+      staked: 1000000000000000000stake
+      mnemonic: ${VALIDATOR2_MNEMONIC}
+  accounts:
+    - name: community
+      coins: 10000000000000000000000basephoton
+      mnemonic: ${COMMUNITY_MNEMONIC}
+    - name: signer1
+      coins: 20000000000000000000000basephoton
+      mnemonic: ${SIGNER1_MNEMONIC}
+    - name: signer2
+      coins: 30000000000000000000000basephoton
+      mnemonic: ${SIGNER2_MNEMONIC}
+
+  genesis:
+    consensus_params:
+      block:
+        max_bytes: "1048576"
+        max_gas: "81500000"
+    app_state:
+      evm:
+        params:
+          evm_denom: basetphoton
+      gov:
+        voting_params:
+          voting_period: "10s"
+        deposit_params:
+          max_deposit_period: "10s"
+          min_deposit:
+            - denom: "basetphoton"
+              amount: "1"
+      transfer:
+        params:
+          receive_enabled: true
+          send_enabled: true
+      feemarket:
+        params:
```

Comment:
do we need to add the other params?

Reply:
It's get merged with default parameters, so only need to specify the differences.